bedrock/mozorg/templates/mozorg/developer/includes/sub-nav.html

```diff
@@ -7,7 +7,26 @@
 {% block sub_nav_class %}technology{% endblock %}
 
 {% block sub_nav_primary_links %}
-<li><a href="{{ url('mozorg.technology') }}" data-link-name="Web Innovations" data-link-type="nav" data-link-position="subnav">{{ _('Web Innovations') }}</a></li>
-<li><a href="{{ url('mozorg.developer') }}" data-link-name="Mozilla Developers" data-link-type="nav" data-link-position="subnav">{{ _('Mozilla Developers') }}</a></li>
-<li><a href="https://developer.mozilla.org" data-link-name="MDN" data-link-type="nav" data-link-position="subnav">{{ _('MDN') }}</a></li>
+<li>
+  {% if 'technology' in request.path %}
+    <a href="{{ url('mozorg.developer') }}" data-link-name="Developer Hub" data-link-type="nav" data-link-position="subnav">{{ _('Developer Hub') }}</a>
+  {% else %}
+    <a href="{{ url('mozorg.technology') }}" data-link-name="Technology Hub" data-link-type="nav" data-link-position="subnav">{{ _('Technology Hub') }}</a>
```

Comment:
Not quite sure where this request came from, but fwiw I thin the subnav shold probably be consistent across all pages. `Technology Hub` isn't a term we use either (the whole `hub` word isn't really used publically).

Reply:
`/technology/` still has a link on every page as it's a primary nav item, so two links are maybe a bit redundant. See what they say, but I honestly got confused when I loaded things up before reading the PR :)

Reply:
:) Yeah, let's see what the feedback there is. Quick to revert that change.

Reply:
If we do keep this link, we should probably make it consistent with the link above and just use "Technology"